types/queries.go

```diff
@@ -105,10 +106,47 @@ type QueryRequest struct {
 	IBC          *IBCQuery          `json:"ibc,omitempty"`
 	Staking      *StakingQuery      `json:"staking,omitempty"`
 	Distribution *DistributionQuery `json:"distribution,omitempty"`
-	Stargate     *StargateQuery     `json:"stargate,omitempty"`
+	Grpc         *GrpcQuery         `json:"grpc,omitempty"`
```

Comment:
I think we still need both of them here because wasmd will need to handle `.Stargate` differently than `.Grpc`. Stargate requires keeping the current behaviour in which the response is JSON serialized. Existing contracts rely on this behaviour.

Overall this is really more an addition of a new query than a rename

Reply:
Ah, true. We want to enforce protobuf here.

Reply:
Fixed this. It is just an addition now.